docs/docs/target/container_image_config.md

```diff
@@ -0,0 +1,42 @@
+# Container Image Configuration
+
+Scan a container image, local or remote. Scans the [configuration](https://github.com/opencontainers/image-spec/blob/2fb996805b3734779bf9a3a84dc9a9691ad7efdd/config.md) that defines the container image (you can examine it using the `docker save` command).
+
+In addition to scanning the configuration of the container image, Trivy can also scan the contents of the image. For more info, see the [Image](./container_image.md) page.
+
+Usage:
+
+```shell
+trivy image --image-config-scanners misconfig,secret localimage:tag
+trivy image --image-config-scanners misconfig,secret myregistry.io/myimage:tag
+```
+
+## Scanners
+
+Supported scanners:
+
+- Misconfigurations
+- Secrets
+
+By default, no image configuration scanners are enabled. You can configure which scanners are used with the [`--image-config-scanners` flag](../configuration/others.md#enabledisable-scanners).
```

Comment:
This link to `--scanners` flag.
So it can confused users.

Reply:
https://github.com/aquasecurity/trivy/pull/8305/commits/6a69bd8567866effb47b76df08e1a25d95259ffd